src/cron/cron.py

```diff
@@ -82,24 +90,35 @@
                     ATTRS_PREFIX + "USER_NB_DOCUMENTS"
                     ]
 
-EMAIL_ATTR_INDEX = 0
-USER_FIRST_LOGIN_INDEX = 3
-USER_LAST_LOGIN_INDEX = 4
-
 def normalize_date(value: date|None) -> str|None:
     return value.strftime('%Y-%m-%d') if value is not None else value
 
 for user in users:
-    user = list(user)
-    email = user.pop(EMAIL_ATTR_INDEX)
-    user[USER_FIRST_LOGIN_INDEX] = normalize_date(user[USER_FIRST_LOGIN_INDEX])
-    user[USER_LAST_LOGIN_INDEX] = normalize_date(user[USER_LAST_LOGIN_INDEX])
+    print(user)
+    email = user["email"]
+    user.pop("email")
+    user["user_first_login"] = normalize_date(user["user_first_login"])
+    user["user_last_login"] = normalize_date(user["user_last_login"])
+    # We enroll users only when they are added to the instance
```

Comment:
You're suggesting that the users should have been added to the database the day of the run, right?

```suggestion
    # We enroll users only when they have just been registered in the instance
```